### ISSUE TITLE
Link thumbnails not working

### ISSUE THREAD

**Issue by Sbrjt** [joelshepherd/tabliss#619]
The link thumbnails for _https://chat.openai.com_ and _https://web.whatsapp.com_ are not working (display as ▶)

![Screenshot (72)](https://github.com/joelshepherd/tabliss/assets/83166133/1773093d-9b63-4bbb-81bb-043a2d1fea9a)

Is it the case with everyone?

**Comment by theerfan** [comment]
Quick question, @Sbrjt, how did you add the other urls with their proper thumbnail?

**Comment by Sbrjt** [comment]
In icons, there's a website icon option. @theerfan 

[See](https://github.com/joelshepherd/tabliss/assets/83166133/aba2a43e-9081-4239-b0df-7455bbcaabea)

**Comment by theerfan** [comment]
Yeah, weirdly enough that never works for me.

**Comment by Sbrjt** [comment]
@theerfan Does it work after importing my settings?

[tabliss.json](https://github.com/joelshepherd/tabliss/files/12168508/tabliss.txt)
(Please change file name to tabliss.json as github didn't allow me to upload json)

**Comment by theerfan** [comment]
Yeah it did, thank you so much!

**Comment by theerfan** [comment]
But I just realized that it has entirely destroyed the Unsplash pictures in the process lol

I guess I just have to try to debug this extension myself to see why things go wrong

**Comment by KayeAye** [comment]
Just had the same problem. Tabliss uses duckduckgo's icon service to retrieve the icons and it appears duckduckgo's copy of the icon is corrupted. I managed a workaround by using css that directly grabs the icon, this does undermine the point of using duckduckgo to grab the icons anonymously though. Here's the CSS for the scispace one (typeset.io). "Alt" was found by inspecting element (it's just the url). The padding is the same as the font size you see for that quick link's .widget div (I believe this is what is controlled by the 'size' slider in the display settings).

```css
img[alt="typeset.io"] {
	font-size:0;
	padding: 18px 18px 0 0;
	background: url('https://scispace.com/static/favicon.svg');
	background-size:cover;
}
```

Edit: The problem with chatgpt (your issue) is actually that duckduckgo still has the default icon in place for the chat.openai.com subdomain. You can use `https://icons.duckduckgo.com/ip3/openai.com.ico`  as the background image url and still avoid directly grabbing the icon from openai

**Comment by Sbrjt** [comment]
Yeah. duckduckgo's favicon grabber is not working for links with subdomains: https://icons.duckduckgo.com/ip3/https://chat.openai.com

Google's utility is working tho:  https://www.google.com/s2/favicons?sz=32&domain_url=https://chat.openai.com.

(in [Display.tsx](https://github.com/joelshepherd/tabliss/blob/main/src/plugins/widgets/links/Display.tsx))

@joelshepherd

**Comment by piotrowskiadam** [comment]
Yes I have the same issue, for example for web version of whatsapp. Also it would be cool if it could grab icon for specific Google services subdomains like https://messages.google.com instead of showing Google icon everywhere.

**Comment by MikeAnita** [comment]
Hello, how do I get a "favicon"
displayed on an internal site?
e.g. 192.168.1.1 would be my FritzBox
Thanks
